server/server.py

```diff
@@ -13,6 +14,7 @@ def check():
     return "🚀"
 
 
+SLACK_SIGNING_SECRET = os.environ["SLACK_SIGNING_SECRET"]
```

Comment:
There is still a chance that everything will crash (though smaller likelihood). I think you should use `os.environ.get("SLACK_SIGNING_SECRET", "")` so that we don't have a key error and can specify a default value.

Reply:
A good point, but I wonder if there's a value in instead simply logging+crashing when `SLACK_SIGNING_SECRET` isn't found (which would, of course, still require that I catch the `KeyError`). After all, it should _always_ be available on launch, and if we specify a default we'll get all kinds of errors later on that might be more difficult to debug.